### PR TITLE
Add client-config-dir support

### PR DESCRIPTION
**Pull request by Scriptkiddi**
Refactor and add the client-config-dir option and support

### CODE REVIEW

File: tasks/client_keys.yml [hunk 1]
```diff
@@ -9,25 +9,25 @@
     mode: 0400
 
 - name: Generate client key
-  command: "openssl req -nodes -newkey rsa:{{ openvpn_rsa_bits }} -keyout {{ item }}.key -out {{ item }}.csr -days {{ openvpn_openssl_days }} -{{ openvpn_openssl_digest }} -subj /CN=OpenVPN-Client-{{  inventory_hostname[:24]  }}-{{  item[:24]  }}/"
+  command: "openssl req -nodes -newkey rsa:{{ openvpn_rsa_bits }} -keyout {{ item.name }}.key -out {{ item.name }}.csr -days {{ openvpn_openssl_days }} -{{ openvpn_openssl_digest }} -subj /CN=OpenVPN-Client-{{  inventory_hostname[:24]  }}-{{  item.name[:24]  }}/"
```
Comment (SlothOfAnarchy) [inline tasks/client_keys.yml]: why `[:24]` ?

File: tasks/config.yml [hunk 2]
```diff
@@ -28,11 +28,23 @@
     group: root
     mode: 0644
 
+- name: Create client-config-dir
+  file:
+    path: "{{ openvpn_base_dir }}/staticclients"
+    state: directory
+
+- name: Configure client-config-dir
```
Comment (SlothOfAnarchy) [inline tasks/config.yml]: I don't think you're configuring the directory

File: tasks/client_keys.yml (hunk 1 above)
Comment (Scriptkiddi) [reply tasks/client_keys.yml]: because the common name can at most be 64 characters

File: tasks/config.yml (hunk 2 above)
Comment (Scriptkiddi) [reply tasks/config.yml]: fixed

**Comment by Scriptkiddi** [comment]
@SlothOfAnarchy please rereview

**Pull request merged**